Doc/c-api/buffer.rst

```diff
@@ -517,6 +517,15 @@ Buffer-related functions
 
    ``0`` is returned on success, ``-1`` on error.
 
+.. c:function:: int PyObject_CopyToObject(PyObject *obj, void *buf, Py_ssize_t len, char fortran)
```

Comment:
`PyBuffer_ToContiguous` uses 'order' instead of 'fortran' and this is probably a better name. You also need to specify what `len` is. And I think it could be clearer if we named `obj <-> dest` and `buf <-> src`.

Reply:
The function name is named according to PEP3118, and the fort parameter is modified to keep it consistent with other functions.

Reply:
However, the name `PyObject_CopyToObject` already explains its purpose, and users who actually use C-API will not be confused.

Reply:
But this does not help. PEP 3118 only exposes the API but the parameter names do not need to be the same. Note that we already use 'order'. What's important is the function name. In C, you don't have keyword arguments.

Reply:
Also, you need to say that the allowed values for 'fort' (or 'order' or whatever its final name is) are 'C', 'F' and 'A'. Even if it's written in the PEP, it is important to indicate in the docs what's the expectation.

Reply:
I'll add documentation about the fortran parameter and describe it in detail.